addons/explosives/functions/fnc_placeExplosive.sqf

```diff
@@ -57,13 +57,18 @@ if (isNumber (_magazineTrigger >> "digDistance")) then {
 
     //Get Surface Type:
     private _canDigDown = true;
-    private _surfaceType = surfaceType _pos;
-    if ((_surfaceType select [0,1]) == "#") then {_surfaceType = _surfaceType select [1, 99];};
-    if ((_surfaceType != "") || {isClass (configFile >> "CfgSurfaces" >> _surfaceType >> "soundEnviron")}) then {
-        private _soundEnviron = getText (configFile >> "CfgSurfaces" >> _surfaceType >> "soundEnviron");
-        TRACE_2("Dig Down Surface",_surfaceType,_soundEnviron);
-        _canDigDown = !(_soundEnviron in ["road", "tarmac", "concrete", "concrete_int", "int_concrete", "concrete_ext"]);
+
+    // If dig distance is negative (=> placed closer towards the sky), ignore digging requirements
+    if (_digDistance < 0) then {
+        private _surfaceType = surfaceType _pos;
+        if ((_surfaceType select [0,1]) == "#") then {_surfaceType = _surfaceType select [1, 99];};
+        if ((_surfaceType != "") || {isClass (configFile >> "CfgSurfaces" >> _surfaceType >> "soundEnviron")}) then {
+            private _soundEnviron = getText (configFile >> "CfgSurfaces" >> _surfaceType >> "soundEnviron");
+            TRACE_2("Dig Down Surface",_surfaceType,_soundEnviron);
+            _canDigDown = !(_soundEnviron in ["road", "tarmac", "concrete", "concrete_int", "int_concrete", "concrete_ext"]);
```

Comment:
unrelated to this PR, but I wonder if we should be using common_canDig here instead of hardcoded array?